os/installing-to-disk.md

```diff
@@ -2,7 +2,7 @@
 
 ## Install script
 
-There is a simple installer that will destroy everything on the given target disk and install CoreOS. Essentially it downloads an image, verifies it with gpg, and then copies it bit for bit to disk.
+There is a simple installer that will destroy everything on the given target disk and install CoreOS. Essentially it downloads an image, verifies it with gpg, and then copies it bit for bit to disk. An installation requires at least 4.5 GB of usable space on the device, although CoreOS doesn't currently utilize all of it.
```

Comment:
It does use all of it though. I thought we specified that the minimum disk disk was 8GB somewhere else. That is the number I would actually like to go with.